src/bp-activity/bp-activity-functions.php

```diff
@@ -4298,6 +4309,17 @@ function_exists( 'bb_moderation_allowed_specific_notification' ) &&
 
 		$send_email = true;
 
+		// Stop sending email notification to user who has muted notifications.
+		if ( $parent_comment->user_id !== $commenter_id ) {
```

Comment:
@kishanbuddyboss Remove this condition. It has been previously stated.

Reply:
Fixed